scripts/tests/chiptest/test_definition.py

```diff
@@ -111,18 +111,16 @@ def wait(self, timeout=None):
                     self.cv_stopped.wait()
 
     def __startServer(self, runner, command):
-        app_cmd = command + ['--interface-id', str(-1)]
```

Comment:
Isn't this going to break running YAML tests on Darwin, by no longer passing that argument?